fpdf/line_break.py

```diff
@@ -75,6 +76,8 @@ def __init__(self):
         #     SpaceHint is used fo this purpose.
         # 3 - position of last inserted soft-hyphen
         #     HyphenHint is used fo this purpose.
+        #     If print_sh=True, soft-hyphen is treated as
```

Comment:
Shouldn't this big comment be converted to a docstring,
so that it gets rendered in the API docs?
https://pyfpdf.github.io/fpdf2/fpdf/

Reply:
I don't think this class is part of the public API, and most of the comment is about its internals.
But the part I added about the `print_sh` parameter should indeed go into a docstring.